### PR TITLE
feat: Add region and account parameters as optional global defaults

### PR DESCRIPTION
**Pull request by edisongustavo**
This lets me create cross-account and cross-region monitoring. Currently only the current account is supported.

My PR adds these parameters at the *global level* because I didn't want to go through every resource and add these extra parameters. This creates a limitation where an instance of `MonitoringFacade` would refer only to those specific region/accounts.

This is good enough for me though, so please advise if it's not desired.

---

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license_

### CODE REVIEW

**Review by echeung-amzn — APPROVED** [review]
This seems like a good way to go about doing it. The logical separation of facades to handle different accounts' metrics seems appropriate.

I'll mark this as `do-not-merge` so that we can get a second review too.

**Review by ayush987goyal — APPROVED** [review]

**Review by svkurowski — APPROVED** [review]
Looks good to me! Thank you for your contribution.

**Pull request merged**
